### PR TITLE
add new path to admin html into server.go

### PR DESCRIPTION
**Pull request by Carlos7979**
Nico, there is a new path into server.go wich serve a new html

### CODE REVIEW

File: admin/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <script src="main.js"></script>
+    <script>
+        window.addEventListener('load', main);
+    </script>
+    <title>Admin | SimpleApp</title>
+</head>
+<body>
+    <p>This is a basic test</p>
+</body>
+</html>
```
Comment (nhatcher) [inline admin/index.html]: Missing new line at the end of the file

File: admin/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (nhatcher) [inline admin/index.html]: What are these two for? We do not have them in the other html files

File: server.go [hunk 3]
```diff
@@ -108,9 +108,17 @@ func mainHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func adminHandler(w http.ResponseWriter, r *http.Request) {
+	path := r.URL.Path[1:]
+	log.Printf("Serving Admin file: %s", path)
+	appPath := fmt.Sprintf("%s", path)
```
Comment (nhatcher) [inline server.go]: you do not need to use this variable, you an use path directly

File: admin/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="main.js"></script>
+    <script>
+        window.addEventListener('load', main);
```
Comment (nhatcher) [inline admin/index.html:8]: main is undefined. you can define an empty main function in `main.js` if you want

**Review by nhatcher — CHANGES_REQUESTED** [review]
Some very small changes but looks good!

File: admin/index.html (hunk 2 above)
Comment (Carlos7979) [reply admin/index.html]: I don't know, it is automatic with the html template of VScode, do you want it to be removed?

File: server.go (hunk 3 above)
Comment (Carlos7979) [reply server.go]: Sure! becase there isn't some text wich to chain to path

File: admin/index.html (hunk 1 above)
Comment (Carlos7979) [reply admin/index.html]: Done. Why is this important?

File: admin/index.html (hunk 2 above)
Comment (nhatcher) [reply admin/index.html]: Yes, let's keep it as small as possible. We can put it back later when we know what that is for :)

File: admin/index.html (hunk 4 above)
Comment (Carlos7979) [reply admin/index.html:8]: All rigth

File: server.go (hunk 3 above)
Comment (nhatcher) [reply server.go]: To be precisse, you can just do:
```javascript
	path := r.URL.Path[1:]
	log.Printf("Serving Admin file: %s", path)
        http.ServeFile(w, r, path)
```

**Review by nhatcher — CHANGES_REQUESTED** [review]

File: admin/index.html (hunk 1 above)
Comment (nhatcher) [reply admin/index.html]: It's good programming practice. The source of some issues and all linters complain. Also see:

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: admin/main.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+function main() {
+    
+}
```
Comment (nhatcher) [inline admin/main.js]: Missing new line :)

**Review by nhatcher — CHANGES_REQUESTED** [review]

File: admin/main.js (hunk 5 above)
Comment (Carlos7979) [reply admin/main.js]: Ok, please, can you explain me why is this important? I would like to learn about this

File: admin/index.html (hunk 2 above)
Comment (nhatcher) [reply admin/index.html]: This is still not done

**Review by nhatcher — CHANGES_REQUESTED** [review]

File: admin/main.js (hunk 5 above)
Comment (nhatcher) [reply admin/main.js]: See comment above

File: admin/index.html (hunk 2 above)
Comment (Carlos7979) [reply admin/index.html]: Ok! I didn't see your answer, thanks for explanation

**Review by nhatcher — APPROVED** [review]
Good to go!

File: admin/main.js (hunk 5 above)
Comment (Carlos7979) [reply admin/main.js]: Thanks, I didn't realize that comments appear above

**Pull request merged**